templates/supertest/post/post.handlebars

```diff
@@ -27,11 +27,15 @@
       .set('{{headerApiKey.type}}', process.env.{{headerApiKey.name}})
       {{/if}}
       {{#is contentType 'application/json'}}
+      {{#if inputs.body}}
+      .send({{printJSON inputs.body.data}})
```

Comment:
in your explanation of your PR you do not use the data object. For me it works if I remove this